test-suite/js/src/test/resources/ScalaJSDefinedTestNatives.js

```diff
@@ -50,4 +50,6 @@
     }
   };
   $g.ConstructorDefaultParam = ConstructorDefaultParam;
+  $g.Foo = {};
+  $g.Bar = {};
```

Comment:
This enters JS' global scope. They should have longer names that won't clash with anything.